src/js/glowingbear.js

```diff
@@ -118,10 +111,10 @@ weechat.controller('WeechatCtrl', ['$rootScope', '$scope', '$store', '$timeout',
     })();
 
     // Show a TLS warning if GB was loaded over an unencrypted connection,
-    // except for local instances (local files, testing, cordova, or electron)
+    // except for local instances (local files, testing)
```

Comment:
I guess electron can remain here?

Reply:
I will make a pull request to remove everything electron too